src/ai_baby_monitor/ui/streamlit_components.py

```
@@ -41,7 +41,7 @@ def display_sidebar(
             st.caption(f"Subsample rate: {selected_config.subsample_rate}")
         st.divider()
         with st.expander("LLM Model"):
-            st.caption(f"LLM Model: {os.getenv('LLM_MODEL_NAME')}")
+            st.caption(f"OpenAI Model: gpt-4o-mini")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)